### ISSUE TITLE
Support npm and jsr

### ISSUE THREAD

**Issue by GuangYiL** [solidjs-community/solid-primitives#726]
### Describe The Problem To Be Solved

deno is getting better with the release of 2.0. I thought it would be nice to add a jsr dependency url to the page after npm

### Suggest A Solution

After building the workflow and testing and verifying it, publish the npm and jsr repositories, and add a new column to the web page to show the jsr address

**Comment by okikio** [comment]
Yeah, simply adding a `deno.jsonc` file to each package directory should be enough, you then just need to  list the direct dependencies in the `imports` field, then copy the `exports` field from `package.json`.

The biggest question mark ends up being slow types. `jsr.io` is optimized to ensure type inference of the user of the library is ultra fast, so jsr strongly enforces explicit types for exported symbols. Looking through the codebase this shouldn't be a particularly big problem tbh, since it looks like most primitives already do this in their source code. Everything should be good to go.

So basically add a root `deno.jsonc` this allows us to take advantage of `deno`'s built in support for jsr workspaces:

```ts
{
  "workspace": [
    "packages/*"
  ],
  "compilerOptions": {
    "strict": true,
    "jsx": "preserve",
    "jsxImportSource": "solid-js",
    "lib": ["DOM", "ESNext"]
  },
  "nodeModulesDir": "auto"
}
```

then we just implement a `deno.jsonc` for each package.

A good example is this for `@solid-primitives/utils`

```ts
{
  "name": "@solid-primitives/utils",
  "version": "6.4.0",
  "exports": {
    ".": "./src/index.ts",
    "./immutable": "./src/immutable/index.ts"
  },
  "publish": {
    "include": [
      "LICENSE",
      "README.md",
      "src/**/*.ts",
      "src/**/*.tsx",
      "package.json"
    ],
    "exclude": [
      "test",
      "dev",
      "dist",
      "tsconfig.json"
    ]
  }
}
```

**Comment by davedbase** [comment]
@GuangYiL first part of support for JSR is in place. We're close to releasing!
